.env.test

```diff
@@ -5,9 +5,9 @@ FLASK_DEBUG=1
 FLASK_SKIP_DOTENV=True
 
 # Change the following values for local development.
-trello_key=kghaksbnsjlaksjdlaksjdldkjlmbsanmbd
-trello_token=asdkjalskjdlaskjdaskjdaskljdlaskjdaskjdaksjdlakjdlaskjdlaskjdlaskjd
-trello_boardid=abcdefg
-todo_list_id=abcdefghijklmnopqrstuvwxyz
-doing_list_id=abcdefghijklmnopqrstuvwxyz
-done_list_id =abcdefghijklmnopqrstuvwxyz
\ No newline at end of file
+trello_key=test
+trello_token=test
+trello_boardid=test
+todo_list_id=test
+doing_list_id=test
+done_list_id=test
```

Comment:
Minor point, but it's slightly nicer to have different values for these things, like `test-token` or `test-list-id` so that e.g. unit tests can check you make HTTP requests with `key=test-key` and `token=test-token`. Or if you're debugging a test you can see at a glance that the expected values are being used